charts/opensearch/ci/ci-ingress-class-name-values.yaml

```diff
@@ -318,12 +320,21 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+startupProbe:
+  enabled: false
+  tcpSocket:
+    port: 9200
+  initialDelaySeconds: 5
+  periodSeconds: 10
+  timeoutSeconds: 3
+  failureThreshold: 30
+
 readinessProbe:
   failureThreshold: 3
-  initialDelaySeconds: 10
+  initialDelaySeconds: 900
```

Comment:
15 minutes is a long time. CI doesn't have an implicit timeout does it?

Reply:
> I think I'm coming into this quite late but one thing I don't see is how authentication is managed through the endpoints? Is the health endpoint now unauthenticated on the Opensearch API?

So @DandyDeveloper previously we were trying to have an authenticated probe but that has a lot of cons which was overweighing the pros. You can follow the discussion [here](https://github.com/opensearch-project/helm-charts/pull/172#issuecomment-1114815558)

Reply:
@DandyDeveloper Are we good to close this comment?